notificationdemo/source/c/CMakeLists.txt

```diff
@@ -15,23 +15,31 @@
 #
 
 cmake_minimum_required(VERSION 2.8.8)
-
-project (C-SDK-sample)
-enable_language(C)
+project (C-SDK-sample LANGUAGES C)
```

Comment:
`LANGUAGES` is not supported in cmake 2.8.8. Use `project(C-SDK-sample C)` instead.